partition/roles/pixiecore/README.md

```diff
@@ -26,4 +26,5 @@ Deploys pixiecore in a systemd-managed Docker container.
 | pixiecore_metal_hammer_logging_cert         |           | set metal-hammer to send logs to a remote endpoint and authenticate with this cert for mtls auth              |
 | pixiecore_metal_hammer_logging_key          |           | set metal-hammer to send logs to a remote endpoint and authenticate with this key for mtls auth               |
 | pixiecore_metal_hammer_logging_tls_insecure |           | set metal-hammer to send logs to a remote endpoint without verifying the tls certificate for mtls auth        |
+| pixiecore_metal_hammer_ntp_servers          |           | A list of custom NTP servers                                                                                  |
```

Comment:
Please add it to the defaults as otherwise deployments will fail if this new variable is not defined.

Consider making it a list and join the elements when passing them to the program.